server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/CassandraRabbitMQJamesServerMain.java

```diff
@@ -20,24 +20,54 @@
 package org.apache.james;
 
 import static org.apache.james.CassandraJamesServerMain.REQUIRE_TASK_MANAGER_MODULE;
+import static org.apache.james.modules.blobstore.BlobStoreChoosingConfiguration.readBlobStoreChoosingConfiguration;
```

Comment:
why a static import?

Reply:
Why not ?

Reply:
because we decided, long ago, to not use static import unless we think methods/values are part of our frequently used "DSL".
We can change our rules but I think it makes reading harder.

Reply:
Anyway it is removed.